content/workshops/ansible_tower/_index.adoc

```diff
@@ -0,0 +1,95 @@
+---
+title: Ansible Tower Workshop
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-magic
+---
+
+:domain_name: redhatgov.io
+:workshop_prefix: company-name
+:tower_url: https://{workshop_prefix}.*student_number*.{domain_name}
+:ssh_url: https://{workshop_prefix}.*student_number*.{domain_name}/wetty/ssh/ec2-user
+
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Automation for Everyone
+
+
+
+.Today's Agenda
+
+- Arrivals
+- Introductions / Agenda
+- Presentation
+- Setup
+- Exercise 1.0 - Ad-Hoc Commands
+- Break
+- Exercise 1.1 - Writing Your First Playbook
+- Exercise 1.2 - Running Your First Playbook
+- Exercise 1.3 - Installing Ansible Tower
+- Exercise 1.4 - Configuring Ansible Tower
+- Exercise 1.4 - Creating and Running a Job Template
+- Wrap Up
+
+
+
+.Before We Get Started
+
+[.lead]
+Overview
```

Comment:
Change this to be a 2nd level heading instead of using `[.lead]`. While using `[.lead]` does look alright when generated, it doesn't make it a proper heading class which renders better when using PatternFly.

`== Overview`